instrumentation/opentelemetry-api/opentelemetry-api-1.31/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/v1_31/metrics/MeterTest.java

```diff
@@ -108,7 +108,7 @@ void longCounter() {
   void doubleCounter() {
     DoubleCounterBuilder builder =
         meter.counterBuilder("test").ofDoubles().setDescription("d").setUnit("u");
-    assertThat(builder).isInstanceOf(ExtendedDoubleCounterBuilder.class);
+    Assertions.assertThat(builder).isInstanceOf(ExtendedDoubleCounterBuilder.class);
```

Comment:
(same question in a few places)
```suggestion
    assertThat(builder).isInstanceOf(ExtendedDoubleCounterBuilder.class);
```